### ISSUE TITLE
make CTRL+SHIFT+C work in terminal

### ISSUE THREAD

**Issue by meysholdt** [gitpod-io/gitpod#183]
**Describe the bug**
In the Terminal View, `CTRL+SHIFT+C` opens the "Review Conversions" View.

**Expected behavior**
In the Terminal View, `CTRL+SHIFT+C` should copy the current selection to the clipboard, just like `CTRL+SHIFT+V` already pasts the contents from the clipboard. `CTRL+C` can not be used because it terminates the current process.

**Comment by purkhusid** [comment]
This would great! Is this something that Gitpod can fix or is this an browser limitation?

**Comment by akosyakov** [comment]
@purkhusid Could you try to switch to VS Code: https://www.gitpod.io/docs/feature-preview/#default-ide and check whether it works there?

**Comment by purkhusid** [comment]
I'm actually using VS Code. This seems like a Firefox issue. I had to change the CTRL + SHIFT + C VS code shortcut to something else and then  copying worked but CTRL + SHIFT + V does not work in Firefox. It seems like pasting i completely broken in the terminal in Firefox. I don't even get the option to paste when right clicking.

When I pasted for the first time in Chrome I was asked to give the tab access to my clipboard. Maybe this is missing in Firefox?
